src/index.html

```diff
@@ -22,6 +22,355 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Logo"
+          class="logo__image"
+        />
+      </a>
+      <nav class="nav">
+        <li class="nav__list">
+          <a
+            href="#"
+            class="nav__link is-active"
+          >
+            Apple
+          </a>
+        </li>
+        <li class="nav__list">
+          <a
+            href="#"
+            class="nav__link"
+          >
+            Samsung
+          </a>
+        </li>
+        <li class="nav__list">
+          <a
+            href="#"
+            class="nav__link"
+          >
+            Smartphones
+          </a>
+        </li>
+        <li class="nav__list">
+          <a
+            href="#"
+            class="nav__link"
+            data-qa="nav-hover"
+          >
+            Laptops & Computers
+          </a>
+        </li>
+        <li class="nav__list">
+          <a
+            href="#"
+            class="nav__link"
+          >
+            Gadgets
+          </a>
+        </li>
+        <li class="nav__list">
+          <a
+            href="#"
+            class="nav__link"
+          >
+            Tablets
+          </a>
+        </li>
+        <li class="nav__list">
+          <a
+            href="#"
+            class="nav__link"
+          >
+            Photo
+          </a>
+        </li>
+        <li class="nav__list">
+          <a
+            href="#"
+            class="nav__link"
+          >
+            Video
+          </a>
+        </li>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="Card"
+          class="card__image"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <div class="card__meta">Product code: 195434</div>
+        <div class="card__rating">
+          <div class="card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+          </div>
```

Comment:
```suggestion
            <div class="stars stars--4 card__stars">
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
            </div>
```
You don't need additional wrapper to add element class.